public/javascript/bot.js

```diff
@@ -99,7 +99,7 @@ function messageHandler(event) {
             handlerFunctions['getPokemon'](userAPI, event.threadID, event.senderID, message);
         } else if ((/^@stock .+$/).test(message)) {
             handlerFunctions['getStock'](userAPI, event.threadID, message);
-        } else if ((/^@weather ([0-9]{5}|([a-zA-Z ]+(, )?[a-zA-Z ]+))$/).test(message)) {
+        } else if ((/^@weather ([0-9]{5} [CF]|([a-zA-Z ]+(, )?[a-zA-Z ]+))$/).test(message)) {
```

Comment:
Yeah like that. `@weather 78701 --unit C`, or `@weather 78701 -u F`, or lower cases. If we use flags, we are guaranteed that the characters after the `-` are not part of the address, so we can simply substring the flag out, and use the `@weather 78701` part of the body the old way.